app/theme/client/imports/components/contextual-bar.css

```diff
@@ -56,9 +56,10 @@
 
 		padding: var(--default-padding);
 
-		border-bottom: solid 1px var(--color-gray-light);
+		margin: 0 -8px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "margin" to come before "padding" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5416119)